apps/prairielearn/assets/scripts/instructorAssessmentStatisticsClient.ts

```diff
@@ -0,0 +1,31 @@
+import { onDocumentReady } from '@prairielearn/browser-utils';
+
+declare global {
+  interface Window {
+    histogram: any;
+    scatter: any;
+    parallel_histograms: any;
+  }
+}
+
+onDocumentReady(() => {
+  document.querySelectorAll('.histogram').forEach((histogramElement) => {
+    const data = JSON.parse(histogramElement.getAttribute('data-histogram') ?? '');
+    const xgrid = JSON.parse(histogramElement.getAttribute('data-xgrid') ?? '');
+    const options = JSON.parse(histogramElement.getAttribute('data-options') ?? '');
+    window.histogram(histogramElement, data, xgrid, options);
+  });
+
+  document.querySelectorAll('.scatter').forEach((scatterElement) => {
+    const xdata = JSON.parse(scatterElement.getAttribute('data-xdata') ?? '');
+    const ydata = JSON.parse(scatterElement.getAttribute('data-ydata') ?? '');
+    const options = JSON.parse(scatterElement.getAttribute('data-options') ?? '');
+    window.scatter(scatterElement, xdata, ydata, options);
+  });
+
+  document.querySelectorAll('.parallel_histograms').forEach((parallelHistogramsElement) => {
```

Comment:
Convention for class names is usually kebab-case (`parallel-histograms`). Also consider the `js-` prefix above.

Reply:
Updated this class name to `kebab-case` and added a `js-` prefix in [00ffe80](https://github.com/PrairieLearn/PrairieLearn/pull/9809/commits/00ffe80fa4857a2dae6d76fbf63b45e21cdcecc2).